### PR TITLE
fix(top-app-bar): Adjust styles for top app bar and drawer.

### PR DESCRIPTION
**Pull request by williamernest**
Adjust styles for top app bar and drawer. 
Update top app bar to 0.35.1.
Replace #2643 fix with #2704 fix for media query.

### CODE REVIEW

File: src/frame/TopAppBarFramePage.js [hunk 1]
```diff
@@ -72,6 +72,18 @@ class TopAppBarFramePage extends Component {
...
   }
 
+  getOffset(type = propToVariant.standard) {
+    let offsetClass = '';
+    if(type === propToVariant.standard || type === propToVariant.fixed) {
+      offsetClass = 'mdc-top-app-bar--fixed-adjust';
+    } else if (type === propToVariant['short-collapsed']) {
+      offsetClass = 'mdc-top-app-bar--short-fixed-adjust'
+    } else {
+      offsetClass = `${type.variant}-fixed-adjust`;
```
Comment (patrickrodee) [inline src/frame/TopAppBarFramePage.js]: Is this missing an extra dash (`${type.variant}--fixed-adjust`) ?

Comment (williamernest) [reply src/frame/TopAppBarFramePage.js]: No, this is correct for the variants.

Comment (patrickrodee) [reply src/frame/TopAppBarFramePage.js]: Dope! Thanks for clarifying.

File: src/frame/TopAppBarFramePage.js [hunk 2]
```diff
@@ -72,6 +72,18 @@ class TopAppBarFramePage extends Component {
     );
   }
 
+  getOffset(type = propToVariant.standard) {
+    let offsetClass = '';
```
Comment (patrickrodee) [inline src/frame/TopAppBarFramePage.js]: Could initialize this as `${type.variant}-fixed-adjust` then drop the else case

**Pull request merged**